### PR TITLE
Add mezzanine

### PR DESCRIPTION
**Pull request by dimasciput**
This PR will add dockerized mezzanine

### CODE REVIEW

**Comment by timlinux** [comment]
Note that the production host will not be k4 (as error your  makefile). @cchristelis to provide host ip

**Comment by timlinux** [comment]
Actually we should not us ip but ssh alias

File: deployment/Makefile [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+#    P R O D U C T I O N     C O M M A N D S
+# ----------------------------------------------------------------------------
+default: web
+run: build permissions web migrate collectstatic
+
+deploy: run
+	@echo
+	@echo "------------------------------------------------------------------"
+	@echo "Bringing up fresh instance "
+	@echo "You can access it on http://localhost:61200"
```
Comment (meomancer) [inline deployment/Makefile]: change 61200 to 63300

**Comment by meomancer** [comment]
@dimasciput 
i see that in makefile, some of description using 61200
that should be changed to 63300?
and also in nginx too

**Comment by meomancer** [comment]
LGMT
mezzanine for this is up
we could make another PR for deployment stuff

**Pull request merged**